app/workers/dependency_resolution_worker.rb

```diff
@@ -29,24 +29,43 @@ def content_item_dependees
                                      dependent_lookup: Queries::GetDependees.new).call
   end
 
-  def present_content_store(dependent_content_id)
+  def downstream_content_item(dependent_content_id)
     scope = ContentItem.where(content_id: dependent_content_id)
-
-    if content_store == Adapters::DraftContentStore
+    if draft?
       latest_content_item = Queries::GetLatest.call(scope).last
     else
-      latest_content_item = ContentItemFilter.new(scope: scope).filter(
-        state: "published",
-      ).last
+      latest_content_item = ContentItemFilter.new(scope: scope).filter(state: "published").last
     end
 
     return unless latest_content_item
 
+    if draft?
+      present_content_store(latest_content_item)
+    else
+      downstream_publish(latest_content_item)
+    end
+  end
+
+  def draft?
+    content_store == Adapters::DraftContentStore
+  end
+
+  def present_content_store(latest_content_item)
     PresentedContentStoreWorker.perform_async_in_queue(
       PresentedContentStoreWorker::LOW_QUEUE,
-      content_store: content_store,
+      content_store: Adapters::DraftContentStore,
```

Comment:
Ah actually had already been done in this commit: https://github.com/alphagov/publishing-api/commit/d198f7eec3634141b9023f35564014864f831df0 

Reason for change was initially we replaced just publishing with a new worker, which then expanded into the various downstream workers.